packages/react-components/react-table/src/components/TableHeaderCell/useTableHeaderCellStyles.styles.ts

```diff
@@ -41,6 +42,10 @@ const useStyles = makeStyles({
       { selector: 'focus-within' },
     ),
     position: 'relative',
+    [`[${KeyboardResizingCurrentColumnDataAttribute}]`]: {
```

Comment:
Please add a visual regression test for this

Reply:
done